cmd/kpod/common.go

```diff
@@ -32,6 +32,16 @@ func getStore(c *libkpod.Config) (storage.Store, error) {
 	return store, nil
 }
 
+func getStoreOptions(c *libkpod.Config) storage.StoreOptions {
```

Comment:
I think we might want to redo this to not just get the storage options, but turn the complete config into a runtime - so don't return the storage options, but make the runtime and return it. That was, we can eventually expand it to parse and handle the entire config.